rmw_opendds_cpp/src/opendds_static_publisher_info.cpp

```diff
@@ -90,5 +110,90 @@ rmw_ret_t OpenDDSStaticPublisherInfo::get_status(
 
 DDS::Entity * OpenDDSStaticPublisherInfo::get_entity()
 {
-  return topic_writer_;
+  return writer_;
+}
+
+void OpenDDSStaticPublisherInfo::cleanup()
+{
+  if (writer_) {
+    writer_ = nullptr;
+  }
+  if (publisher_) {
+    publisher_ = nullptr;
+  }
+  OpenDDSPublisherListener::Raf::destroy(listener_);
+  callbacks_ = nullptr;
+}
+
+OpenDDSStaticPublisherInfo::OpenDDSStaticPublisherInfo(DDS::DomainParticipant_var dp
+  , const rosidl_message_type_support_t & ros_ts
+  , const char * topic_name
+  , const rmw_qos_profile_t & rmw_qos
+  )
```

Comment:
```suggestion
  , const rmw_qos_profile_t & rmw_qos)
```

Reply:
I think the purpose of putting a comma in front of a line is to avoid changes to  unaffected lines when one or more lines are later changed (added, removed or reordered). In this case rmw_qos is the last parameter now. If we add one later, the rmw_qos line stays the same in git repo. Moving ')' to the end of rmw_qos cancels the purpose of a front comma. With that said, I removed the 2 spaces before ')' so it can be a clear separator between the parameters and the initializations.